components/bl-list-component/src/list.js

```diff
@@ -0,0 +1,11 @@
+import { ListContent } from "./list-content";
+
+export function List({ props, items }) {
+  const ComponentType = props.type;
+
+  return (
+    <ComponentType className="list">
+      <ListContent props={props} items={items} />
```

Comment:
There is no need for List Content component

`{ items.map(item => (<Item .../>)) }`